physionet-django/physionet/settings/base.py

```diff
@@ -31,6 +31,7 @@
 ENABLE_LIGHTWAVE = config('ENABLE_LIGHTWAVE', default=True, cast=bool)
 SSO_REMOTE_USER_HEADER = config('SSO_REMOTE_USER_HEADER', default='HTTP_REMOTE_USER')
 SSO_LOGIN_BUTTON_TEXT = config('SSO_LOGIN_BUTTON_TEXT', default='Login')
+PRIVACY_POLICY_HTML = config('PRIVACY_POLICY_HTML')
```

Comment:
Could we add some a default=None (or similar) here, and then omit the checkbox on the registration page in this case?